docs/prerequisites.md

```diff
@@ -0,0 +1,39 @@
+
+# Prerequisites
+## Install dependencies
+
+- Python 3.7+
+- pip
+
+## Create the bot
+
+This bot runs on your own machine or server. You'll need to create the bot and invite it to your Discord server before you can run it.
+1. Create a bot on the [Discord Developer Portal](https://discordapp.com/developers/)
+2. Enable **Message Content Intent** from the Bot tab. 
+3. Navigate to **OAuth2 > URL Generator** Make sure the bot has `Send Messages`, `Read Messages/View Channels`, `Read Message History` and `Embed Links` permissions.
+4. Invite the bot to the server
+
+## Prepare data directory and config file
+
```

Comment:
The recommended approach to setting configuration values is through a configuration file. The following instructions will guide you to running the bot for the first time, using file-based configuration.

Reply:
Is this inteded to be within the suggestion?

Reply:
Yep lol